TS02-Convert-to-Date-Time-Class-POSIX.Rmd

```diff
@@ -86,22 +87,12 @@ library(lubridate)  #work with dates
 #set working directory to ensure R can find the file we wish to import
 #setwd("working-dir-path-here")
 
```

Comment:
Importing 15min data is in Lesson 01, so that is why I pulled it. I see what you are saying about each lesson being standalone code. What I did in later lessons is include in the r chunk comments "If you didn't complete Lesson 01, load the file here:"